cookbook/serializer.py

```diff
@@ -200,20 +202,36 @@ class Meta:
 
 class KeywordSerializer(UniqueFieldsMixin, serializers.ModelSerializer):
     label = serializers.SerializerMethodField('get_label')
+    image = serializers.SerializerMethodField('get_image')
+    numrecipe = serializers.SerializerMethodField('count_recipes')
 
     def get_label(self, obj):
         return str(obj)
 
+    def get_image(self, obj):
+        recipes = obj.recipe_set.all().exclude(image__isnull=True).exclude(image__exact='')
```

Comment:
i think we should add a `space=self.request-space` filter here so that in the unlikely Szenario that a keyword is added to a foreign recipe no data is leaked when returning the space image.